config/ban.php

```diff
@@ -24,5 +24,17 @@
     */
 
     'load_default_migrations' => true,
-
+   
+    /*
+    |---------------------------------------------------------------------------
+    | RURL to redirect banned user to
+    |---------------------------------------------------------------------------
+    |
+    | Provide a url, this is where a banned user will be redirected to 
```

Comment:
Some URL example values (absolute, relative, route) in PHPDoc will make it more useful.

Reply:
Added examples